app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/AMLSController.scala

```diff
@@ -112,4 +116,38 @@ class AMLSController @Inject()(
       }
     }
 
+  import play.api.data.{Form, FormError}
+
+  private def formWithRefinedErrors(form: Form[AMLSForm]): Form[AMLSForm] = {
```

Comment:
can we move this method to  uk.gov.hmrc.agentsubscriptionfrontend.controllers.AMLSForms package object?